c2corg_ui/static/js/documentediting.js

```diff
@@ -504,6 +504,22 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
   app.utils.pushToArray(object, property, value, event);
 };
 
+/**
+ * @param {number} value
+ * @export
+ */
+app.DocumentEditingController.prototype.convertExposureTime = function(value) {
+  if (value === 0) {
+    this['scope']['image']['exposure_time_converted'] = this.alerts.gettext('cannot be 0');
```

Comment:
The alert service should be used to display alerts, not to translate strings.
Anyway I don't think it the place for adding an error message, it should be done in the form itself using the usual validation & message tools, as for the other attributes.